recipe/meta.yaml

```diff
@@ -76,6 +76,8 @@ test:
     - h5py
   commands:
     - pip check
+    - OMPI_MCA_plm=isolated mpiexec -np 1 python -c "import h5py;h5py.run_tests()"  # [mpi != 'nompi']
+    - python -c "import h5py;h5py.run_tests()"  # [mpi == 'nompi']
```

Comment:
We need `sys.exit` for these to actually raise the exit code correctly. That said, why can't we leave these in `run_test.py`?

Reply:
that was my attempt to make pytest aware that this is an MPI build.